src/Driver/solver_configs.jl

```diff
@@ -0,0 +1,115 @@
+# CLIMA solver configurations
+#
+# Contains helper functions to establish solver configurations to be
+# used with the CLIMA driver.
+
+"""
+    CLIMA.SolverConfiguration
+
+Parameters needed by `CLIMA.solve!()` to run a simulation.
+"""
+struct SolverConfiguration{FT}
+    name::String
+    mpicomm::MPI.Comm
+    dg::DGModel
+    Q::MPIStateArray
+    t0::FT
+    timeend::FT
+    dt::FT
+    forcecpu::Bool
+    numberofsteps::Int
+    init_args
+    solver
```

Comment:
Do may want to comment on what these are?

Reply:
It's more or less a repeat of the parameters to `setup_solver()`. I don't know what the protocol is in such a case...